src/main/telemetry/frsky.h

```diff
@@ -22,10 +22,13 @@ typedef enum {
     FRSKY_VFAS_PRECISION_HIGH
 } frskyVFasPrecision_e;
 
+typedef void frSkyTelemetryInitFrameFunc(void);
```

Comment:
Oops, I wasn't thinking clearly there. Revised comment: standard suffix for function typedef is `Fn`, so this should be `frSkyTelemetryInitFrameFn`.